pages/osx/compgen.md

```diff
@@ -1,6 +1,7 @@
 # compgen
 
 > A built-in command for auto-completion in bash, which is called on pressing TAB key twice.
+> More information: <https://www.cyberciti.biz/open-source/command-line-hacks/compgen-linux-command/>.
```

Comment:
There isn't an `osx` man page available, because there isn't an `osx` version. The `linux` and `osx` page are exactly the same. But let's leave this for now, I'll take care of it when I make a PR to move them to common.
Edit: same for `feh` and `hostname`.